### PR TITLE
Carmen Cow Dance Show

### PR DESCRIPTION
**Pull request by CJWZhang**
`_If` any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Comment by CJWZhang** [comment]
thanks!

File: solutions/usaco-688.mdx [hunk 1]
```diff
@@ -57,4 +57,73 @@ for j in pers:
...
+```java
+
+import java.io.*;
+import java.util.*;
+
+public class HPS {
+	public static void main(String[] args) throws Exception {
+
+		
+	    Scanner in = new Scanner(new File("hps.in"));
```
Comment (SansPapyrus683) [inline solutions/usaco-688.mdx:71]: don't use scanner

File: solutions/usaco-688.mdx [hunk 2]
```diff
@@ -57,4 +57,73 @@ for j in pers:
...
+	    
+	    PrintWriter out = new PrintWriter(new File("hps.out"));
```
Comment (SansPapyrus683) [inline solutions/usaco-688.mdx:73]: put this @ the bottom where you actually use it

File: solutions/usaco-690.mdx [hunk 3]
```diff
@@ -160,4 +161,82 @@ public class CowDanceShow {
 	
 </JavaSection>
 
+```java
+import java.io.*;
+import java.util.*;
+
+public class cowdance {
+
+	static int Tmax;
```
Comment (SansPapyrus683) [inline solutions/usaco-690.mdx:170]: use camelcase

File: solutions/usaco-690.mdx [hunk 4]
```diff
@@ -160,4 +161,82 @@ public class CowDanceShow {
...
+		
+		
+	
+		out.println(k);
+		in.close();
+		out.close();
+
+	}
+	
+	static boolean check(int k) {
```
Comment (SansPapyrus683) [inline solutions/usaco-690.mdx:214]: check what?

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/usaco-688.mdx (hunk 1 above)
Comment (CJWZhang) [reply solutions/usaco-688.mdx:71]: yeah, got it, i will change it. This is from a long time ago, and I was just testing it out. Thanks

**Comment by CJWZhang** [comment]
fixed it, thanks

**Comment by SansPapyrus683** [comment]
image
read these

**Comment by CJWZhang** [comment]
Hey sorry, where do I find this?

Thank you!

**Comment by SansPapyrus683** [comment]
it's in the pr template that popped up when you made this pr

**Comment by CJWZhang** [comment]
Got it, thanks. I will take a look at it soon.

**Pull request closed without merging**